### PR TITLE
Cleaned data

### PR DESCRIPTION
**Pull request by katesanders9**

### CODE REVIEW

File: tasks/.DS_Store
Comment (yeganehkordi) [inline tasks/.DS_Store:1]: We can drop `.DS_Store` files.

File: tasks/Commonsense Morality-Text Label Validate-Collect-Extended/updated_batch.csv
Comment (yeganehkordi) [inline tasks/Commonsense Morality-Text Label Validate-Collect-Extended/updated_batch.csv:1]: What do you think about dropping `Answer.text_judgment_1`?

File: tasks/Generics Embodiment Pilot - gpt3-gens grp4 batch2/updated_batch.csv [hunk 1]
```diff
@@ -0,0 +1,1984 @@
+﻿Title,Description,Keywords,example,generic,always_generic,only_generic,ex_iloc,file_name,ex_num,pattern_num,pnum_gpt3,ppl_gpt3,need_anns,Answer.always_generic,Answer.ee,Answer.is_tricky,Answer.only_generic
```
Comment (yeganehkordi) [inline tasks/Generics Embodiment Pilot - gpt3-gens grp4 batch2/updated_batch.csv]: We can drop `Answer.ee`.

File: tasks/IMDB Sentiment Completions (RLUE) 5/updated_batch.csv [hunk 2]
```diff
@@ -0,0 +1,304 @@
+﻿Title,Description,Keywords,prompt,machine_completion,algorithm,Answer.coherence,Answer.ee,Answer.sentiment
```
Comment (yeganehkordi) [inline tasks/IMDB Sentiment Completions (RLUE) 5/updated_batch.csv]: Let's drop `Answer.ee`

File: tasks/NER - Task scruples 26,200 - 30,922/updated_batch.csv
Comment (yeganehkordi) [inline tasks/NER - Task scruples 26,200 - 30,922/updated_batch.csv:1]: I think we can drop `Answer.te`.

File: tasks/OOF Hits 11/updated_batch.csv
Comment (yeganehkordi) [inline tasks/OOF Hits 11/updated_batch.csv:1]: we can drop `Answer.te`.

File: tasks/Pseudoword Dataset Creation PPDB/updated_batch.csv [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
...ying and neutering them before adoption -- even at a very <strong><font color=""blue"">early</strong></font> age -- outweighs any health risks .","worst <strong><font color=""blue"">thing</strong></font> about being an actress : `` crying scenes . ...","meyer and gaston kent , northrop grumman vice president of investor relations , rebutted repeated assertions by boeing and members of the congressional delegation of washington state -- where boeing has major facilities -- about <strong><font color=""blue"">too</strong></font> many jobs going overseas .","the membership action plans that ukraine and georgia are seeking from this summit would <strong><font color=""blue"">put</strong></font> them on a path toward membership but would not guarantee it .","the wfp has budgeted $ 2.9 billion this year -- all from donor nations -- to conduct its feeding programs around the world , including <strong><font color=""blue"">large</strong></font> efforts in sudan , somalia , ethiopia and other nations that could not otherwise feed themselves .",8,24,1,33,19,8,17,4,1,5,23,33,2,21,21,50,1,35,14,25,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,lobby,NONE
```
Comment (yeganehkordi) [inline tasks/Pseudoword Dataset Creation PPDB/updated_batch.csv]: Most of these answers are NONE. Are they correct?

File: tasks/Step 4 Selecting good - bad answers to questions (short)/updated_batch.csv
Comment (yeganehkordi) [inline tasks/Step 4 Selecting good - bad answers to questions (short)/updated_batch.csv:1]: Do we have all of these fields in the HTML file?

File: tasks/Summarization (RLUE) 1/updated_batch.csv
Comment (yeganehkordi) [inline tasks/Summarization (RLUE) 1/updated_batch.csv:1]: We can drop `,Answer.ee`

File: tasks/TuringAdvice_GPT3-instruct/updated_batch.csv
Comment (yeganehkordi) [inline tasks/TuringAdvice_GPT3-instruct/updated_batch.csv:1]: We can drop `Answer.te`

File: tasks/defeasible inference RUN6 (500)/updated_batch.csv
Comment (yeganehkordi) [inline tasks/defeasible inference RUN6 (500)/updated_batch.csv:1]: Can we drop incomplete rows?

**Review by yeganehkordi — COMMENTED** [review]
Looks good to me.
Please rename the files to `batch.csv` because their name has been hardcoded in the codes.

File: tasks/Step 4 Selecting good - bad answers to questions (short)/updated_batch.csv
Comment (katesanders9) [reply tasks/Step 4 Selecting good - bad answers to questions (short)/updated_batch.csv:1]: Yes, in the CSV HTML

File: tasks/Pseudoword Dataset Creation PPDB/updated_batch.csv (hunk 3 above)
Comment (katesanders9) [reply tasks/Pseudoword Dataset Creation PPDB/updated_batch.csv]: I believe so

File: tasks/defeasible inference RUN6 (500)/updated_batch.csv
Comment (katesanders9) [reply tasks/defeasible inference RUN6 (500)/updated_batch.csv:1]: The rows all look complete to me - some involve not answering the first question and providing an explanation instead

**Comment by katesanders9** [comment]
Updated branch based on feedback

File: tasks/Commongen Evals (RLUE) 2/batch_updated.csv
Comment (yeganehkordi) [inline tasks/Commongen Evals (RLUE) 2/batch_updated.csv:1]: We have both `batch_updated.csv` and `batch.csv` in this directory. Which one is the final version?

**Review by yeganehkordi — COMMENTED** [review]
Thank you.
Can you please drop the `.DS_Store` files and then we are ready to merge this pr.

**Comment by katesanders9** [comment]
Removed .DS_Store files

File: tasks/Commongen Evals (RLUE) 2/batch_updated.csv
Comment (katesanders9) [reply tasks/Commongen Evals (RLUE) 2/batch_updated.csv:1]: Resolved

**Pull request merged**